src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
```

Comment:
What if you need to create 100 getRandomBalls? You will also create  Ball ball1 = Lottery.getRandomBall();